### PR TITLE
feat: namespace controller creates the rabc needed for the job

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it
feat: namespace controller creates the rabc needed for the job

#### Which issue(s) this PR fixes

Fix https://github.com/kubeagi/arcadia/issues/629

### CODE REVIEW

**Review by bjwswang — APPROVED** [review]

File: deploy/charts/arcadia/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: arcadia
 description: A Helm chart(KubeBB Component) for KubeAGI Arcadia
 type: application
-version: 0.2.25
+version: 0.2.26
```
Comment (bjwswang) [inline deploy/charts/arcadia/Chart.yaml:5]: @0xff-dev please rebase to latest

**Review by Abirdcfly — APPROVED** [review]

File: controllers/base/namespace_controller.go [hunk 2]
```diff
@@ -194,16 +204,41 @@ func (r *NamespaceReconciler) checkSkippedNamespace(ctx context.Context, namespa
...
+	if err := r.Client.Create(ctx, &sa); err != nil && !errors.IsAlreadyExists(err) {
+		logger.Error(err, "failed to create serviceaccount", "ServiceAccount", evaluation.RAGJobServiceAccount, "Namespace", instance.Name)
+		return err
+	}
+	clusterRoleBindingName := env.GetString(evaluation.RAGClusterRoleBindingEnv, evaluation.RAGJobClusterRoleBinding)
+	crb := v1.ClusterRoleBinding{}
+	if err := r.Client.Get(ctx, types.NamespacedName{Name: clusterRoleBindingName}, &crb); err != nil {
+		return err
+	}
+	crb.Subjects = append(crb.Subjects, v1.Subject{
```
Comment (nkwangleiGIT) [inline controllers/base/namespace_controller.go:238]: seems we just cover the add of subject when create NS, but didn't cover deletion of ns, and remove subject of deleted NS.

File: config/rbac/rag_job_rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (nkwangleiGIT) [inline config/rbac/rag_job_rbac.yaml:1]: seems this file is useless? as we will create default clusterrole during helm install

File: pkg/evaluation/jobs.go [hunk 4]
```diff
@@ -39,6 +39,11 @@ const (
 	defaultPVCMountPath = "/data/evaluations"
 	defaultTestRagFile  = "ragas.csv"
 	defaultMCImage      = "kubeagi/minio-mc:RELEASE.2023-01-28T20-29-38Z"
+
+	RAGClusterRoleBindingEnv = "RAG_EVAL_CLUSTERROLEBINDING"
```
Comment (nkwangleiGIT) [inline pkg/evaluation/jobs.go:43]: add some comments to explain env and name of CRB and SA

**Pull request merged**